Competition2018/src/main/java/competition/subsystems/offboard/packets/ElevatorPositionCommandPacket.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.offboard.packets;
+
+import org.apache.log4j.Logger;
+
+public class ElevatorPositionCommandPacket {
+	
+	private static Logger log = Logger.getLogger(ElevatorPositionCommandPacket.class);
+	
+	public final int commandId;
+	public final ElevatorGoal elevatorGoal;
+	
+	public enum ElevatorGoal {
```

Comment:
Can we build this into the elevator subsystem rather than defining our own enum here? It would be nice to be able to specify heights everywhere in the Java system. Right now, this enum just has to be converted back into a height using another manual if statement.

Reply:
okie